app/shared/availability-view/TimelineGroups/TimelineGroup/AvailabilityTimeline/Reservation.js

```diff
@@ -47,8 +47,11 @@ function Reservation({ onClick, ...reservation }) {
       {reservation.comments && <div><Glyphicon glyph="comment" /> {reservation.comments}</div>}
     </Popover>
   );
+  const reservationLinkClassName = reservation.comments ?
+    'reservation-link with-comments' :
+    'reservation-link';
   return (
-    <Link className="reservation-link" onClick={() => onClick && onClick(reservation)}>
+    <Link className={reservationLinkClassName} onClick={() => onClick && onClick(reservation)}>
```

Comment:
- [ ] You should use [classnames](https://github.com/JedWatson/classnames) here so this would be simpler.

So something like:
```javascript
className={classNames('reservation-link', { 'with-comments': reservation.comments })}
```